saleor/graphql/webhook/mutations/webhook_trigger.py

```diff
@@ -84,6 +84,21 @@ def validate_event_type(cls, event_type, object_id):
                 code=WebhookTriggerErrorCode.INVALID_ID,
             )
 
+    @classmethod
+    def _get_async_event_or_raise_error(cls, event_type):
+        event = ASYNC_WEBHOOK_TYPES_MAP.get(event_type)
+        if event:
+            return event
+        event_name = event_type[0].upper() + to_camel_case(event_type)[1:]
```

Comment:
This is going to trigger an `IndexError` if `event_type` is None. This case was covered in line 68 previously.

Reply:
this validation is done after the grapql subscription query validation is done (the event name came from there) so there should be no case where we end with empty_event name, hovewer I will add a safe method to handle case when event_name is None or empty string